src/pages/Companies.js

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect } from 'react';
+import { FaMicrophone } from 'react-icons/fa';
+import { IoMdSettings, IoIosArrowBack } from 'react-icons/io';
+import { useDispatch, useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { fetchCompanies } from '../redux/Companies';
+import '../styles/Company.css';
+
+const Companies = () => {
+  const companies = useSelector((state) => state.companies);
+  const urlElement = window.location.href.split('/')[4];
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchCompanies(urlElement));
+  }, [urlElement, dispatch]);
+  return (
+    <>
+      <nav className="navBar">
+        <NavLink to="/"><IoIosArrowBack /></NavLink>
+        <p>Companies</p>
+        <div className="settings">
+          <FaMicrophone />
+          <IoMdSettings />
+        </div>
+      </nav>
+      <div className="companies-container">
+        {companies.map((company) => (
+          <div key={company.symbol} className="company-field">
+            <NavLink to={`/companies/${urlElement}/${(company.symbol).replace(/\s/g, '')}/company-details`}>
+              <div className="div-company">
+                <div>
+                  <div className="name-market">
+                    <em>{company.companyName}</em>
+                    <em>{`Market Cap: $${(company.marketCap)}`}</em>
+                  </div>
+                </div>
+              </div>
+            </NavLink>
+          </div>
+        ))}
+      </div>
```

Comment:
- [ ] The same thing here, kindly check my comment above ⬆
> Kindly, update the design for the companies page to look similar to the original design 💯

| Your Design  | Original Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/205421328-8e6ea3f8-508b-4cfb-8cdc-e143be8f0a6e.png" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/205421352-c01cfe04-959b-4637-9dad-62b4bc649e9f.png" width="500" /> |

Reply:
Just did the same! Thank you!